wp-content/themes/haruoshi_origin/src/scss/archive-works.scss

```
@@ -41,3 +41,56 @@
     object-fit: cover; //IE11では使えないよ
   }
 }
+
+.pagination {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+  &__page-numbers {
+    font-size: 20px;
+    margin-right: 14px;
+    width: 20px;
+    display: inline-block;
+    &--active {
+      position: relative;
+    }
+    &--active::after {
+      content: "";
+      width: 20px;
+      height: 2px;
+      display: inline-block;
+      background-color: $main-color;
+      position: absolute;
+      bottom: -4px;
+      left: calc(50% - 10px);
+    }
+  }
+  &__pre {
+    margin-top: 4px;
+    margin-right: 20px;
+  }
+  &__next {
+    margin-top: 4px;
+  }
+  &__arrow {
+    content: "";
+    width: 14px;
+    height: 14px;
+    border-top: solid 2px $main-color;
+    border-right: solid 2px $main-color;
+    position: absolute;
+    &--left {
+      transform: rotate(-135deg);
+    }
+    &--right {
+      transform: rotate(45deg);
+    }
+  }
```

もしかして `pagination__pre` とか `pagination__next` ってクラスはつける必要なくてこれで十分だったりする？
spanはインライン要素だから `display: block;` してあげる必要がありそう。
あと擬似要素じゃなくなったから `content: "";` は要らない。
`position: absolute;` も外せそうだけど、動かしてみたわけじゃないからわからない。absoluteは謎挙動の原因になりやすいから使わなくていいなら使いたくない。
```suggestion
  &__arrow {
    display: block;
    width: 14px;
    height: 14px;
    border-top: solid 2px $main-color;
    border-right: solid 2px $main-color;
    margin-top: 4px;
    &--left {
      transform: rotate(-135deg);
    }
    &--right {
      transform: rotate(45deg);
      margin-right: 20px;
    }
  }
```

---

いけた https://github.com/oooshima/origin_wordpress/pull/35/commits/830d8777c5cb513dcf95b69f936043fd6d73bbef